### PR TITLE
HTML FORM DEVELOPMENT

### PR DESCRIPTION
**Pull request by thankgodr**
Project Summary.

Completed the markup design using HTML and CSS

Install Sticler for linting

Passed all the Stickler checks

### CODE REVIEW

**Comment by shshamim63** [comment]
## Status :ballot_box_with_check:  [Changes Required] :ballot_box_with_check: 
___
Hi @thankgodr  and @sipe-daniel :wave: ,
Files are organized, code looks neat and clean. Almost cloned the actual webpage. But still some space to cover to get approval for this project.
___
### Please follow the instructions below :point_down:  to complete this project.
___
### :sparkle: __Mandatory Changes__ :sparkle: 
1. You need to add a descriptive readme. Without a descriptive readme a project can not get the appearance of a professional project.
2. Stickler is not configured properly. Check this [link](https://github.com/microverseinc/linters-config/tree/master/css) again. Specially No.1 and 2. Make sure that you have enabled stickler and also installed it.
3. Some items are overflowing the page. So please inspect the page and resolve the issue
image
4. The footer text of the form looks too big and it does not look like the original one.
image
5. The input type of phone is 'tel' not text.
https://github.com/sipe-daniel/HTML-Forms/blob/14fda5879c0101f8907714995f1d56940bc31a57/index.html#L65
6. The text should rely on one line and it has more space to its right. Please compare it with the actual one. Which should render at the right most side.
image
7. PR(Pull Request) should contain a summary about the task you have covered so far on this PR. But I cannot see that.
___
### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Comment by thankgodr** [comment]
Project Summary.

- Completed the markup design using HTML and CSS

- Install Sticler for linting

- Passed all the Stickler checks

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!
### Nice job! WOW, your project looks! :+1: 

### But it needs a few changes before can be approved.

Kindly recommended checking HTML markup with this validator https://validator.w3.org/#validate_by_upload.
screen
 * Now it shows ERRORs in your project





### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project still has errors in the HTML file.

File: css/style.css
Comment (Anna-Myzukina) [inline css/style.css]: Line number 77 Value Error : font-size only 0 can be a unit. You must put a unit after your number : 10

Comment (thankgodr) [reply css/style.css]: Hello @anna-Myzukina, Thanks for the feedback. Issue resolved

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

But still some changes need to be made:
* [ ] This project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] Validator still show one error, but it can be fixed by adding `<meta charset="UTF-8">` to `head` of your project.
* [ ] You are using Bootstrap in this project, that's not allowed. Please remove link to Bootstrap file and finish styling using just css.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by pelzolga123** [comment]
## Great work!
### You implemented almost everything, one more thing is left:
 * [ ] Please remove link to Bootstrap file and use plain css for styling. You will have opportunity to work with bootstrap later in program. Before that all HTML/CSS projects should be done only using css, unless requirements ask for Bootstrap. 

### Kindly fix this issue and submit for another code review.
If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by rOluochKe** [comment]
@thankgodr and @Anna-Myzukina Good job on the project, you have done impressive work so far. Though make these changes and resubmit for a code review.

- Use plain css, don't use bootstrap for this project as my @pelzolga123  mentioned in the last review

css

- Your HTML code has an error with the validator

validator

- Fix all these and resubmit for review.
If you are having a problem, reach out for help.

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Hi Team,
## Project looks great. Congratulation :+1: Approved.
Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**
